packages/cluster-ui/README.md

```diff
@@ -17,3 +17,68 @@ export default props => (
   </div>
 );
 ```
+# Development Workflow with CRDB or CockroachCloud
+
+If you would like changes in this package to be reflected in a locally running DB Console instance, you will need to locally link the library to your DB Console project like this:
+
+In `packages/cluster-ui` run:
+
+```
+> yarn
+> yarn link
+> yarn run build:watch
+```
+
+Then in `pkg/ui` in CRDB repo run:
+
+```
+> yarn link "@cockroachlabs/cluster-ui"
+> yarn
+> cd ../../
+> make ui-watch TARGET=http://localhost:8080
+```
+
+While running a local CRDB cluster. This will link your DB Console code to the local copy of `cluster-ui` and the `build:watch` task in `cluster-ui` will update the build with any changes you make which will get picked up in the `ui-watch` job on the DB side.
+
+# Storybook
+
+Learn more about Storybook here: https://storybook.js.org/
+
+You can run storybook for this project using:
+```
+> yarn run storybook
+```
+And opening http://localhost:6006/ in your browser.
+
+The use of storybook in this project is to provide a controlled environment where compontents can be rendered or showcased for testing and sharing purposes outside of the entire DB Console or CockroachCloud app. 
+
+Often we build a component that behaves or looks differently in different scenarios with different inputs. Storybook lets us render the component in all the "interesting configurations" we define so that we can quickly get visual feedback on our work and ensure that we're not breaking functionality when editing code. Otherwise we waste lots of time getting our app in the "right state" to try and see if some random feature works or not.
+
+Components and page developers are encouraged to use Storybook to showcase components and pages in many common states to be able to quickly review their  work visually and to aid others in understanding what the various states pages and components we want to support.
+
+Storybook stories **should** use the CSF format as described here: https://storybook.js.org/docs/react/api/csf in order to facilitate writing unit tests with the storybook components.
+
+# Updating Protobuf definitions from CRDB
+
+The `cluster-ui` components rely on protobuf types and code from CRDB in order to know what data shapes they should expect and how to query them. Sometimes you might need to develop components using updated definitions that were merged recently or are in progress.
+
+The protobuf definitions are currently manually published from the CRDB repo into this package: https://www.npmjs.com/package/@cockroachlabs/crdb-protobuf-client
+
+### How to publish protobuf client updates to npm
+
+Once you have updated definitions locally, run `make ui-lint` to generate them and make sure typescript is run, then modify the `pkg/ui/src/js/package.json` file with a new version (if your work isn't merged yet, use `0.0.<next>-beta.<integer>) like `0.0.23` if the latest one is `0.0.22` and run:
```

Comment:
There is a missing backtick after `<integer>` which is messing up your formatting.

```suggestion
Once you have updated definitions locally, run `make ui-lint` to generate them and make sure typescript is run, then modify the `pkg/ui/src/js/package.json` file with a new version (if your work isn't merged yet, use `0.0.<next>-beta.<integer>`) like `0.0.23` if the latest one is `0.0.22` and run:
```

I think we could use a more explicit example of a prerelease tag here. So instead of `0.0.23` maybe `0.0.23-beta.1`.

Reply:
done. thx!